conda/recipes/libkvikio/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
     {% endif %}
     - ninja
     - {{ stdlib("c") }}
+    - aws-sdk-cpp
```

Comment:
```suggestion
```
@KyleFromNVIDIA do we `aws-sdk-cpp` here? Asking because `libcufile-dev` isn't need?